tests/tickets/16820/manual/ie10hrline.md

```diff
@@ -0,0 +1,11 @@
+@bender-tags: selection, tc, 4.7.1, 16820
+@bender-ckeditor-plugins: wysiwygarea,toolbar,horizontalrule,newpage
```

Comment:
I think `newpage` dependency is not needed. You may just initialize the test with empty editor which will also simplify the test to 2 steps :)